packages/libs/deer/README.md

```diff
@@ -0,0 +1,86 @@
+# Deer
+
+`deer` is a backend agnostic deserialization framework, featuring meaningful error
+messages and context (utilizing [`error-stack`](https://crates.io/crates/error-stack)) and
+a fail-slow behavior by default.
+
+## Fail-Slow
+
+Currently, Rust deserializers are good at one thing: parsing a lot of data correct and
+_fast_. This is often what is desired, but the need for speed sacrifices usability in
+scenarios where multiple errors are of interest.
+`deer` solves this exact problem, by allowing multiple errors, trying as hard as possible
```

Comment:
We might explicitly preface this by saying something to the effect of "`deer` approaches deserialization differently, trading off an acceptable amount of speed to enable the surfacing of multiple errors".

Reply:
changed in [2bb0b1b](https://github.com/hashintel/hash/pull/1163/commits/2bb0b1b818abf9c9c3622249777410805a730abc), is this better?